tcms/testruns/tests/test_admin.py

```diff
@@ -22,3 +28,20 @@ def test_regular_user_can_not_change_testrun(self):
         response = self.client.get(reverse('admin:testruns_testrun_change',
                                            args=[self.test_run.pk]))
         self.assertRedirects(response, reverse('testruns-get', args=[self.test_run.pk]))
+
+    def test_regular_user_can_not_delete_testrun(self):
+        response = self.client.get(reverse('admin:testruns_testrun_delete',
+                                           args=[self.test_run.pk]))
+        self.assertRaisesMessage(response, ('Permission denied: TestRun does not belong to you'))
+
+    def test_superuser_can_delete_testrun(self):
+        self.client.login(
+            username=self.superuser.username,
+            password='password')
+        response = self.client.get(reverse('admin:testruns_testrun_delete',
+                                           args=[self.test_run.pk]))
+        self.assertContains(response, ("Yes, I'm sure"))
```

Comment:
This message is marked for translation so you will have to assert on the translated text as well. That's why we're testing with French & Slovenian translations and you get a failure for Slovenian. Search for the gettext_lazy function.